.eslintrc.json

```diff
@@ -0,0 +1,57 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2021": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest"
+    },
+    // custom rules for supporting VCS default formatter
+    "rules": {
+        "quotes": [
+            "error",
+            "single"
+        ],
+        // windows linebreak style
+        "linebreak-style": [
+            "error",
+            "windows"
+        ],
+        // disallows semicolon in the end of line
+        "semi": [
+            "error",
+            "never"
```

Comment:
just my opinion, better use "always" instead "never"

Reply:
According to the doc: https://eslint.org/docs/latest/rules/semi , I need to use "never" in case of disallowing semicolon

Reply:
ok, let's leave all eslint rules at the discretion of the developer.

Reply:
Sarcasm :) 
I have updated some rules